client/state/sites/blog-stickers/reducer.js

```diff
@@ -17,6 +17,18 @@ export const items = createReducer(
 				[ action.payload.blogId ]: action.payload.stickers,
 			} );
 		},
+		[ SITES_BLOG_STICKER_ADD ]: ( state, action ) => {
+			const { blogId, stickerName } = action.payload;
+
+			// If the blog already has this sticker, do nothing
+			if ( state[ blogId ] && includes( state[ blogId ], stickerName ) ) {
```

Comment:
includes can handle a null first arg, so you can drop the first half of this if statement